SecurityPkg/Test/Mock/Include/GoogleTest/Ppi/MockTcgPpiLib.h

```diff
@@ -0,0 +1,117 @@
+/** @file MockTcgPpiLib.h
+  Google Test mocks for TcgPpi
+
+  Copyright (c) Microsoft Corporation.
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+**/
+
+#ifndef __MOCK_TCG_PPI_H__

```

Comment:
```suggestion
#ifndef MOCK_TCG_PPI_H__

```

leading underscores are reserved for compiler use